Sources/TZSettingsView/TZSettingsList/TZSettingsSection.swift

```diff
@@ -60,10 +60,14 @@ extension TZSettingsView.Section {
         var color: Color
         /// The font to be used in the themed view or section.
         var font: Font
+        /// The font to be used in the themed view or section.
+        var horizontalPadding: CGFloat?
 
-        public init(color: Color, font: Font) {
+        public init(color: Color, font: Font,
+                    horizontalPadding: CGFloat? = 20) {
```

Comment:
I don't think `horizontalPadding` should be optional and having a non-nil default value. I recommend to be non-optional and keep the 20 pt default value